test/inductor/test_cpu_repro.py

```diff
@@ -43,9 +43,12 @@
 run_and_get_cpp_code = test_torchinductor.run_and_get_cpp_code
 TestCase = test_torchinductor.TestCase
 aten = torch.ops.aten
+check_model = test_torchinductor.check_model
 
 
 class CPUReproTests(TestCase):
+    common = check_model
```

Comment:
Other cases in this file can be simplified with `common` too.

Reply:
let's do it at the next step.